src/orquestra/sdk/_base/cli/_dorq/_repos.py

```diff
@@ -133,6 +159,88 @@ def get_wf_outputs(self, wf_run_id: WorkflowRunId, config_name: ConfigName):
 
         return outputs
 
+    def _get_wf_def_model(
+        self, wf_run_id: WorkflowRunId, config_name: ConfigName
+    ) -> WorkflowDef:
+        """
+        Raises:
+            orquestra.sdk.exceptions.NotFoundError: when the wf_run_id doesn't match a
+                stored run ID.
+            orquestra.sdk.exceptions.ConfigNameNotFoundError: when the named config is
+                not found in the file.
+        """
+        try:
+            wf_run = sdk.WorkflowRun.by_id(wf_run_id, config_name)
+        except (exceptions.NotFoundError, exceptions.ConfigNameNotFoundError):
+            raise
+
+        status_model = wf_run.get_status_model()
+        wf_def = status_model.workflow_def
+
+        # TODO: check when this is optional and add a message
```

Comment:
Issue: hanging TODO. Either fix or create new Jira ticket and attach it here

Reply:
Updated!